srv/salt/ceph/mds/pools/default.sls

```diff
@@ -7,27 +7,29 @@ prevent empty rendering:
 
 {% else %}
 
+{# create cephfs_data pool iif it doesn't exist and no fs instance exists #}
 cephfs data:
   cmd.run:
     - name: "ceph osd pool create cephfs_data 256"
-    - unless:
-      - "rados lspools | grep -q cephfs_data"
-      - "ceph fs ls | grep -q ^name"
+    - onlyif:
+      - 'test -z "$(rados lspools | grep cephfs_data)"'
+      - 'test -z "$(ceph fs ls | grep ^name)"'
```

Comment:
Is this "unless" an "or" or an "and"? 
It would be nice if the "ceph osd pool create cephfs_data" is NOT executed in case the pool exists but the ceph fs filesystem is not.

Reply:
We use `onlyif` here and that only executes the command if both tests return true. See https://docs.saltstack.com/en/latest/ref/states/requisites.html#onlyif